src/styles/blocks/card.scss

```diff
@@ -0,0 +1,93 @@
+.card {
+  padding: 16px;
+  width: 166px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__image {
+    margin: 16px 0 37px 3px;
+    height: 134px;
+    width: 160px;
+  }
+
+  &__name {
+    margin-bottom: 4px;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $color-main-accent;
+  }
+
+  &__product-code {
+    margin-bottom: 16px;
+    font-weight: 400;
+    font-size: 10px;
+    font-style: normal;
+    line-height: 14px;
+    color: $color-secondary;
+  }
+
+  &__reviews-number {
+    width: 53px;
+    font-weight: 300;
+    font-size: 10px;
+    line-height: 14px;
+    text-align: right;
+    color: $color-main-accent;
+  }
+
+  &__container-price {
+    margin-bottom: 16px;
+    display: flex;
+    justify-content: space-between;
+    width: 100%;
+  }
+
+  &__price {
+    width: 31px;
+    font-weight: 400;
+    font-size: 12px;
+    line-height: 18px;
+    color: $color-secondary;
+  }
+
+  &__price-number {
+    width: 50px;
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 18px;
+    text-align: right;
+    color: $color-main-accent;
+  }
+
+  &__buy-button {
+    margin: auto;
+    padding: 11px 13px;
+    display: block;
```

Comment:
Pay attention to CSS properties order
https://9elements.com/css-rule-order/
Check yourself everywhere